server/app/services/export/PdfExporter.java

```diff
@@ -313,17 +313,45 @@ private byte[] buildProgramPdf(
       document.add(
           new Paragraph(
               "Admin description: " + programDefinition.adminDescription(), SMALL_GRAY_FONT));
+      document.add(
+          new Paragraph(
+              "Admin short description: "
+                  + programDefinition.localizedShortDescription().getDefault(),
+              SMALL_GRAY_FONT));
       document.add(
           new Paragraph(
               "Time of export: "
                   + timeCreated.format(DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm a")),
               SMALL_GRAY_FONT));
       document.add(new Paragraph("Origin of export: " + baseUrl, SMALL_GRAY_FONT));
 
+      document.add(Chunk.NEWLINE);
+      document.add(new LineSeparator());
+      Paragraph applicationSteps = new Paragraph();
+      programDefinition.applicationSteps().stream()
+          .forEach(
+              step -> {
+                applicationSteps.add(
+                    new Paragraph(
+                        step.getTitle().getDefault() + " : " + step.getDescription().getDefault(),
+                        SMALL_GRAY_FONT));
+              });
+      if (!applicationSteps.isEmpty()) {
```

Comment:
Will we have extra empty lines if there are no application Steps? It would be nice to test that also.

Reply:
Program's cannot be without application steps. It has become a requirement.

Reply:
Should we assert that then or remove the if condition?

Reply:
you are correct. I removed the if{}..

Reply:
@swatkat1 Legacy programs will not have application steps and this will error. We might want to do a follow-up PR to add this back in.

Reply:
Program migration also fails if the importing program doesn't have application steps. I think it is ok to fail as it will help the admins know that Application steps are missing, and they need to add it. Let us discuss it further in slack.